packages/models/src/lib/implementation/schemas.ts

```diff
@@ -140,9 +142,11 @@ export function packageVersionSchema<TRequired extends boolean>(options?: {
 }
 
 /** Schema for a weight */
-export const weightSchema = nonnegativeIntSchema.describe(
-  'Coefficient for the given score (use weight 0 if only for display)',
-);
+export const weightSchema = z
+  .union([nonnegativeIntSchema, nonnegativeFloatSchema])
```

Comment:
Lighthouse `is-crawlable` audit [weights are floating numbers now](https://github.com/GoogleChrome/lighthouse/blob/main/changelog.md#seo-category-reorganization).